lib/matplotlib/backend_managers.py

```diff
@@ -74,6 +73,38 @@ def __init__(self, canvas):
         self.keypresslock = widgets.LockDraw()
         self.messagelock = widgets.LockDraw()
 
+    @property
+    def manager(self):
+        """FigureManager that owns this ToolManager"""
+        return self._manager
+
+    @property
+    def canvas(self):
+        """Canvas managed by FigureManager"""
+        return self._figure.canvas
+
+    @property
+    def figure(self):
+        """Figure that holds the canvas"""
+        return self._figure
+
+    def set_figure(self, figure):
```

Comment:
Ok, I do this as soon as I get home, I have to leave now.

Thanks for the feedback, keep it coming, I want this merged really bad